test/auth_test.py

```diff
@@ -208,35 +212,133 @@ def test_transfered_ongoing_non_continues_ops(cfg):
     assert ticket.transferred() == 200
 
 
-@pytest.mark.benchmark
-def test_run_operation_benchmark(cfg):
-    # Run 1000000 operations with 4 concurrent threads.
-    ticket = Ticket(testutil.create_ticket(ops=["read"]), cfg)
-    operations = 10**6
-    workers = 4
-    chunk = 10**9
-    step = chunk * workers // operations
+class Client:
 
-    def worker(offset, size):
-        while offset < size:
-            ticket.run(Operation(offset, step))
-            offset += step
+    name = "client"
+    workers = 1
 
-    start = time.monotonic()
+    def run(self):
+        start = time.monotonic()
+        threads = []
+        try:
+            for i in range(self.workers):
+                t = util.start_thread(self._worker, name=f"{self.name}/{i}")
+                threads.append(t)
+        finally:
+            for t in threads:
+                t.join()
```

Comment:
maybe add some timeout?

Reply:
I don't think it worth the effort. The benchmarks do not run in the CI, and when you run them locally you can easily interrupt the run in needed. What can make sens i using the pytest-timeout plugin.

I think the next step is to enable the benchmarks in the CI, to make sure they do not
break, and enable pytest timeout for the benchmarks.

Reply:
Turns out we already enable pytest-timeout with 30 seconds timeout per test, so the only thing missing is to enable the benchmarks also in the CI.

We can create special configuration with small number of iterations just to keep the benchmarks running, but I'm not sure it worth the effort.